source/PreferencesPanel.cpp

```diff
@@ -561,10 +565,10 @@ void PreferencesPanel::DrawPlugins()
 				top.Y() += sprite->Height() + 10.;
 			}
 			
-			WrappedText wrap(FontSet::Get(14));
-			wrap.SetWrapWidth(230);
-			static const string empty = "(No description given.)";
-			wrap.Wrap(it.second.empty() ? empty : it.second);
+			WrappedText wrap(font);
+			wrap.SetWrapWidth(MAX_TEXT_WIDTH);
```

Comment:
The panels in the settings screen are 250 pixels wide, designed to have a 10 pixel margin for the text on each side. This would reduce the right margin to 5 pixels, putting the description text off-center.

Reply:
Oops, missed that 230->235 switch.
Addressed.